### ISSUE TITLE
proot-distro Backup

### ISSUE THREAD

**Issue by xx-Anonymous** [termux/termux-packages#7074]

**Comment by WMCB-Tech** [comment]
Backing up Distribution
```
proot-distro backup [distro_name] --output /sdcard/pd_backup.tar.gz
```

Restoring the Distribution
```
proot-distro restore /sdcard/pd_backup.tar.gz
```

**Comment by xx-Anonymous** [comment]
Hey, I using `nethunter` as my `proot-distro` and I'am having trouble backing it up.

I tried the `backup` switch  the `proot-distro` script ,  but I notice it's using `gz` format/extension (I think).
Because when I used the switch and directed the output with `--output downloaded alot of tools and I have less than 500MB in storage.

> Backing up Distribution
> 
> ```
> proot-distro backup [distro_name] --output /sdcard/pd_backup.tar.gz
> ```
> 
> Restoring the Distribution
> 
> ```
> proot-distro restore /sdcard/pd_backup.tar.gz
> ```

Wow was still typing.
Thanks.

I was worried about storage usage from the backup file as `gz` format uses more storage than `xz` format.As I thought I can't write the tar file to my sdcard as restoring may be weird.(was thinking of dropping the `xz` backup file in `~/../usr/var/lib/proot-distro/dlcache`

But makes sense.
Thanks again.

**Comment by xx-Anonymous** [comment]
Hi, I tried the above command, and it actually errored for me.
Here's the output:
![Screenshot_20210629-133244](https://user-images.githubusercontent.com/76891810/123790613-d30b3b00-d8de-11eb-9f20-e8cebe226886.png)
However I tried this and directed it to internal storage and it worked.
Iam using sdcard as internal storage, but I know Termux only uses external storage for extending storage maybe that's the problem?
